### ISSUE TITLE
Identify an acceptable way of running Release CLI in Source-Build release pipeline

### ISSUE THREAD

**Issue by oleksandr-didyk** [dotnet/source-build#3720]
Currently in `Stage-DotNet` the Release CLI is used directly from code (i.e. `dotnet run **/*.proj`) rather then being built, packaged and then consumed as a versioned tool by the release pipeline.

We should investigate and figure out which of the two ways to run the CLI best suits the Source-Build release pipeline.

**Comment by oleksandr-didyk** [comment]
Agreed offline to go with running the Release CLI tool directly from code for the time being (specifically until we are actively developing the source-build release pipeline and interacting with the tool in question)

As such, removing the issue from the pre 9.0 effort

**Comment by oleksandr-didyk** [comment]
Since we are still actively changing the `ReleaseCLI` tooling for issues planned for `Preview 2` the current way of running the tool is advantageous.

Moving to a later `9.0` timeframe

**Comment by MichaelSimons** [comment]
[Triage] There is limited ROI here.
